### ISSUE TITLE
HOW TO SLIDE?

### ISSUE THREAD

**Issue by HoGGamer** [nZombies-Time/nZombies-Rezzurrection#49]
i gotten my shit fixed and now i dont know how to setup sliding. i cant ask discord cause i was kicked for no reason? please answer this devs cause i want back in discord group.

**Comment by Hu55-Hu55** [comment]
Just Download The Quick Slide Addon

**Comment by HoGGamer** [comment]
I dont know if it will screw up diving

**Comment by Hu55-Hu55** [comment]
Yeah....no it just works diffrent i think you can change the quick slide input inside the console
